templates/CRM/Admin/Form/Setting/Localization.tpl

```diff
@@ -37,7 +37,7 @@
                 <span class="description">{$settings_fields.inheritLocale.description}</span>
               </td>
             </tr>
-          {if empty($form.languageLimit)}
+          {if array_key_exists('languageLimit', $form)}
```

Comment:
This looks !(not wrong).

But actually it looks like it was also previously checking the wrong var - it should be uiLanguages.

Reply:
So isset is problematic but array_key_exists is ok for the smarty default escape mode? Interesting.

Reply:
@demeritcowboy yep - I discovered that `$form` is actually assigns as an array to the tpl layer - I can't find where right now

Reply:
Ok so it's changed from a negative to a positive which changes the display logic. Now I'm trying to figure out if uiLanguages is related to languageLimit, i.e. was the original `if` statement intended as display logic or was it a guard against notices.

It's confusing. These fields have the same title.

Reply:
ok - it' some weird logic - but I've fixed it so it should go back to no-change

Reply:
Yeah the original appears to be about display logic - languageLimit only gets set for multilingual, and for single the field is a different field uiLanguages, so the `if`'s are saying only show one.